quicklogic/README.md

```diff
@@ -54,14 +85,26 @@ The most important stages are:
     Runs VPR placement stage. Design placement is stored in the `top.place` file. IO placement constraints for VPR are written to the `top_io.place` file.
 
 - **route**
-    Runs VPR routing. Design routing data is stored in the `top.route` file.
+    Runs VPR pack, place and route flow. The packed design is written to the `top.net` file. design placement and routing data is stored in the `top.place` and `top.route` files respectively. IO placement constraints for VPR are written to the `top_io.place` file.
 
 - **analysis**
     Runs VPR analysis, writes post-route netlists in BLIF and Verilog format plus an SDF file with post routing timing analysis.
 
 - **fasm**
     Generates the FPGA assembly file (a.k.a. FASM) using the routed design. The FASM file is named `top.fasm`.
 
+- **bit**
+    Generates a binary bitstream file from the FASM file using the `qlfasm.py` tool. The bitstream is ready to be loaded to the FPGA.
+
+- **jlink**
```

Comment:
There should be a note added that `jlink`, `openocd` and `bit_v` targets are available only for the `pp3` family.

Reply:
Added a note jlink, openocd and bit_v targets